FAQ.md

```diff
@@ -0,0 +1,111 @@
+  ```
+
+This will take a while ...
+
+## Interesting commands while an upgrade runs
+
+Check the status of your cluster operators:
+```
+oc get co
+```
+
+Check the status of your nodes (upgrade may include an upgrade to Fedora CoreOS):
+```
+oc get nodes
+```
+
+## How can I find out what's inside of a (CI) release and which commit id each component has?
+This one is very helpful if you want to know, if a certain commit has landed in your current version:
+
+  ```
+  oc adm release info registry.svc.ci.openshift.org/origin/release:4.4  --commit-urls
+  ```
+
+  ```
+  Name:      4.4.0-0.okd-2020-04-10-020541
+  Digest:    sha256:79b82f237aad0c38b5cdaf386ce893ff86060a476a39a067b5178bb6451e713c
+  Created:   2020-04-10T02:14:15Z
+  OS/Arch:   linux/amd64
+  Manifests: 413
+
+  Pull From: registry.svc.ci.openshift.org/origin/release@sha256:79b82f237aad0c38b5cdaf386ce893ff86060a476a39a067b5178bb6451e713c
+
+  Release Metadata:
+    Version:  4.4.0-0.okd-2020-04-10-020541
+    Upgrades: <none>
+
+  Component Versions:
+    kubernetes 1.17.1
+    machine-os 31.20200407.20 Fedora CoreOS
+
+  Images:
+    NAME                                           URL
+    aws-machine-controllers                        https://github.com/openshift/cluster-api-provider-aws/commit/5fa82204468e71b44f65a5f24e2675dbfa0f5c29
+    azure-machine-controllers                      https://github.com/openshift/cluster-api-provider-azure/commit/832a43a30d7f00cd6774c1f5cd117aeebbe1b730
+    baremetal-installer                            https://github.com/openshift/installer/commit/a58f24b0df7e3699b39d4ae1d23c45672706934d
+    baremetal-machine-controllers
+    baremetal-operator
+    baremetal-runtimecfg                           https://github.com/openshift/baremetal-runtimecfg/commit/09850a724d9290ffb05db3dd7f4f4c748b982759
+    branding                                       https://github.com/openshift/origin-branding/commit/068fa1eac9f31ffe13089dd3de2ec49c153b2a14
+    cli                                            https://github.com/openshift/oc/commit/2576e482bf003e34e67ba3d69edcf5d411cfd6f3
+    cli-artifacts                                  https://github.com/openshift/oc/commit/2576e482bf003e34e67ba3d69edcf5d411cfd6f3
+    cloud-credential-operator                      https://github.com/openshift/cloud-credential-operator/commit/446680ed10ac938e11626409acb0c076edd3fd52
+    ...
+
+  ```
+
+## What to do in case of errors ?
+If you experience problems during the installation or afterwards, collect data of your cluster with:
+
+```
+oc adm must-gather
+```
+
+A directory with lots of information will be created. Tar zip it and and it to the developers.
+
+Cloud provider secrets, ... will not be included.
+
+Upload it to a file hoster and send the link to the developers (Slack channel, ...)
+
+# External tips for OKD 4
+
+* slack:
+
+  Workspace: Kubernetes, Channel: #openshift-dev
```

Comment:
I'd prefer to have user conversations in `#openshift-user` and have `-dev` for developer communication. Lets mention both

Reply:
Done